packages/main/src/Slider.hbs

```diff
@@ -0,0 +1,38 @@
+<div 
+	id="{{this._id}}"
+	?disabled= "{{disabled}}"
+	value = "{{value}}"
+	label = "{{label}}"
+	?expanded= "{{_expanded}}"
+	@click = {{_onclick}}
+	@keydown={{_onkeydown}}
+	class = "ui5-slider"
+	data-sap-focus-ref
+	@focusin="{{_onfocusin}}"
+	@focusout="{{_onfocusout}}"
+	tabindex="0"
+>
+	<div class="ui5-slider-header-block">
+		<div id="{{this._id}}--label" class="ui5-slider-label"> {{label}}</div>
+		<div class="ui5-slider-invisible-text"></div>
+		<ui5-button class="ui5-slider-arrow" icon="arrow-top" @click={{_onArrowUp}} tabindex="-1"></ui5-button>
+	</div>
+	<div id="{{this._id}}--inner" class="ui5-slider-inner">
+	<div id="{{this._id}}--selection-frame" class="ui5-slider-selection-frame">
```

Comment:
The ui5-slider-selection-frame seems misplaced in "compact", add "sapUiSizeCompact" to the body tag and open the picker.

Reply:
Done